### ISSUE TITLE
Accessors for OrderedList attributes?

### ISSUE THREAD

**Issue by bpj** [nichtich/Pandoc-Elements#44]
Shouldn't there be setters/getters for `start_number`, `numbering_style` and `delimiter`?

BTW I noticed that OrderedList isa AttributesRole. That's hardly correct since its attributes are different.

I'll fix this if you agree.

**Comment by nichtich** [comment]
Yes, good catch. I'd prefer names `startnum` (as used at <http://pandoc.org/MANUAL.html#ordered-lists>), `style`, `delimiter`. For convenience one might also be able to set style and delimiter by string e.g. "i" (LowerRoman), ")" (OneParen), or "i)" (set both) but this may better go into constructor of the type keywords.

**Comment by bpj** [comment]
@nichtich, is this still not fixed?  I'm so ashamed -- one and a half year!
